README.md

```diff
@@ -17,17 +17,20 @@ Thank you for your interest in contributing to the Login.gov IdP! For complete i
 
 ### Installing on your local machine
 
-This installation method is meant for those who are familiar with setting up local development environments on their machines.
+This installation method is meant for those who are familiar with setting up local development environments on their machines. If you encounter errors, see the [Troubleshooting](#troubleshooting) section at the bottom of this README.
 
+We recommend using [Homebrew](https://brew.sh/), [rbenv](https://github.com/rbenv/rbenv), [nvm](https://github.com/nvm-sh/nvm) or other version management tooling to install the below dependencies; while we don't anticipate changing these frequently, this will ensure that you will be able to easily switch to different versions as needed.
+
+#### Dependencies
 1. To start, make sure you have the following dependencies installed and a working development environment:
 
-- Ruby ~> 3.0.3
+- Ruby ~> 3.0.4
 - [PostgreSQL](http://www.postgresql.org/download/)
 - [Redis 5+](http://redis.io/)
 - [Node.js v14](https://nodejs.org)
+-- (to install Node.js v.14 using brew: `brew install node@14`)
```

Comment:
Should we be providing these Homebrew instructions if we're also still recommending nvm for managing Node versions? My expected recommendation for installing Node.js would be through nvm (`nvm install 14`, or just `nvm install` since we have [`.nvmrc`](https://github.com/18F/identity-idp/blob/main/.nvmrc)).